cmd/lncli/neutrino_active.go

```diff
@@ -225,6 +225,51 @@ func getCFilter(ctx *cli.Context) error {
 	return nil
 }
 
+var unbanPeerCommand = cli.Command{
+	Name:        "unbanPeer",
```

Comment:
s/unbanPeer/unbanpeer

Reply:
Thanks just that unban and peer are two separate words so I was following the convention

Reply:
yeah but see how the other commands are done, for example `addpeer`